biogeochem/EDPhysiologyMod.F90

```diff
@@ -1301,18 +1302,18 @@ subroutine SeedIn( currentSite, bc_in )
 
           litt => currentPatch%litter(el)
           do pft = 1,numpft
-             
              ! Seed input from local sources (within site)
              litt%seed_in_local(pft) = litt%seed_in_local(pft) + site_seed_rain(pft)/area
              
              ! Seed input from external sources (user param seed rain, or dispersal model)
              seed_in_external =  seed_stoich*EDPftvarcon_inst%seed_suppl(pft)*years_per_day
-             
+             if(hlm_use_fixed_biogeog.eq.itrue)then 
```

Comment:
should this be: ?
```suggestion
             if ( hlm_use_fixed_biogeog .eq. itrue .and. currentSite%use_this_pft(pft) .eq. ifalse ) then 
```

Reply:
yeah, I agree, @ckoven , it does look like that should have the PFT filter.

Reply:
So, this is the part that breaks the carbon balance at restart. In particular, we aren't actually setting any of the use_this_pft or area_pfts after the model has restarted. Fixing this is the first thing on my to-do list after this set of changes. I think it's best to have both the variables on the restart file, to be consistant with how everything else about the site is set up, rather than recalculating them from the bc_in variable. Yes? No?

Reply:
It sounds like, following a restart, this routine is being called before use_this_pft is getting filled from the bc_in structure.  I think it would probably be a good idea to put the use_this_pft filter in the restart, as its memory overhead is small compared to our cohort variables.

Reply:
On which note, I'm following 'recrate' around as an example of a similarly-dimensioned variable to figure out how to put it in the restart file. I am confused about why it is registered as a cohort variable here...
 
https://github.com/NGEET/fates/blob/82691fbcf1bb49dff83b091acfa452f73166b5be/main/FatesRestartInterfaceMod.F90#L1003

Reply:
Also, as a matter of general code development strategies, would you typically make a new branch to add such a thing to an open PR, or just push more commits to the same branch?

Reply:
This is just my strategy, but 95% of the time I just push new commits to the same branch.  Sometimes I will pack a set of changes into a new branch and use a PR to update the branch with the original pull request, but I only do that usually in two scenarios, 1) to make it clear to reviewers what I changed per their requests, and only if the changes are significant, or 2) if I plan to make a large set of complicated changes to the original feature branch, and want a quick and easy way to abort if I totally mess things up.